pypureclient/flashblade/FB_2_4/models/nfs_export_policy_post.py

```diff
@@ -36,8 +36,7 @@ class NfsExportPolicyPost(object):
         'is_local': 'bool',
         'location': 'FixedReference',
         'policy_type': 'str',
-        'rules': 'list[NfsExportPolicyRuleInPolicy]',
-        'version': 'str'
```

Comment:
No clue how `version` snuck in here in our last release, but it's readonly. Anyone who happens to be setting it in a POST for some reason is already failing. Removing it shouldn't cause any issues.